rust/gui-client/src-tauri/src/client/gui/system_tray.rs

```diff
@@ -134,7 +154,34 @@ pub(crate) enum Icon {
 
 impl Default for Icon {
     fn default() -> Self {
-        Self::Busy
+        Self {
+            base: IconBase::Busy,
+            update_ready: false,
+        }
+    }
+}
+
+impl Icon {
+    fn tauri_icon(&self) -> tauri::Icon {
+        let layers = match self.base {
+            IconBase::Busy => &[LOGO_GREY_BASE, BUSY_LAYER][..],
+            IconBase::SignedIn => &[LOGO_BASE][..],
+            IconBase::SignedOut => &[LOGO_GREY_BASE, SIGNED_OUT_LAYER][..],
```

Comment:
```suggestion
            IconBase::Busy => [LOGO_GREY_BASE, BUSY_LAYER],
            IconBase::SignedIn => [LOGO_BASE, &[]],
            IconBase::SignedOut => [LOGO_GREY_BASE, SIGNED_OUT_LAYER],
```

Out of curiosity, would this work too?

Reply:
Yeah. I could make a special case in the compositor that a blank byte string is just skipped.